src/planscape/planscape/settings.py

```diff
@@ -240,7 +240,9 @@
 SITE_ID = 1
 ACCOUNT_AUTHENTICATION_METHOD = "email"
 ACCOUNT_EMAIL_REQUIRED = True
-ACCOUNT_EMAIL_VERIFICATION = "mandatory"
+ACCOUNT_EMAIL_VERIFICATION = str(
+    config("ACCOUNT_EMAIL_VERIFICATION", default="mandatory")
```

Comment:
I'm not making use of this for this PR, and I thought I'd just remove it, but decided this would be handy for later.